fe1-web/model/objects/__tests__/Chirp.test.ts

```diff
@@ -5,92 +5,94 @@ import {
 } from '../Chirp';
 import { Timestamp } from '../Timestamp';
 import { Hash } from '../Hash';
+import { PublicKey } from '../PublicKey';
 
 const TIMESTAMP = new Timestamp(12345);
 const HASH_ID = new Hash('id');
 const HASH_PARENT = new Hash('parent');
+const PK = new PublicKey('publicKey');
+const TEXT = 'text';
 
 describe('Chirp object', () => {
-  it('can do a state round trip', () => {
+  it('does a state round trip correctly', () => {
```

Comment:
No need to change anything, this is good enough.

However, as a general rule, it's a "harder" problem to do `object->plain data->object` than `plain data->object->plain data` , because the `object->plain data` conversion can be lossy. As such, round-tripping with objects is often a tougher test on the code (and more realistic with respect to the application's needs)

Reply:
Ok, seems fair. I'll remember that. Actually I just wanted to change the name of the test to make it clearer